joomla-projects/gsoc22_accessibility-plugin-script
# Implemented Line Height

### Summary of changes
Users can increase/decrease line height by clicking on the Increase/decrease line height.

### Testing instructions
Just check you are able to increase/decrease lineheight. Do not forget to check for persistent session and reset button.
